packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -35,20 +35,52 @@ export enum SiteSlot {
 }
 
 export interface ISiteManagedClasses {
+    site__headerTitle: string;
```

Comment:
I know the `__` adheres to BEM standards, but so far we haven't been doing this. Wondering what other people think about it...

Reply:
I would expect `site_header_title` for consistency

Reply:
@chrisdholt do we want to follow this convention for all style names that current exist in this project? Happy to update them all now.

Reply:
Let's update them if it's quick. Otherwise, you can create a task to do this in another PR as it was pre-existing. We'll just want to make sure we do the clean up on it asap.